docs/UserManuals/ConfigUI/Jenkins.md

```diff
@@ -24,7 +24,18 @@ For help on Username and Password, please see Jenkins docs on [using credentials
 Click `Test Connection`, if the connection is successful, click `Save Connection` to add the connection.
 
 ### Step 2 - Setting Data Scope
-There is no data cope setting for Jenkins.
+
+![image](https://user-images.githubusercontent.com/3294100/199530047-787b25d6-696a-4545-860c-9738d85a0076.png)
+
+#### Jobs
+
+Choose the Jenkins jobs to collect.
+
+#### Data Entities
+
+Jenkins only support only one entity `CI/CD`.
```

Comment:
Text Change:
"Jenkins only supports `CI/CD` domain entities, transformed from Jenkins builds and stages."